logrecorder-assertions/src/main/kotlin/io/github/logrecorder/assertion/blocks/AbstractMessagesAssertionBlock.kt

```diff
@@ -34,9 +35,10 @@ abstract class AbstractMessagesAssertionBlock : MessagesAssertionBlock {
     override fun addExpectation(
         logLevelMatcher: LogLevelMatcher,
         messageMatchers: List<MessageMatcher>,
-        propertyMatchers: List<PropertyMatcher>
+        propertyMatchers: List<PropertyMatcher>,
```

Comment:
Classname doesnt fit anymore
eg: AbstractLogRecordAssertionBlock

Reply:
As separately talked about, this would be a breaking change as well as the `Message` part of the name not actually being a reference to only asserting the message part of a log message/entry.